spec/integrations/triggers_spec.rb

```diff
@@ -226,3 +226,468 @@
     end
   end
 end
+
+describe "Logidze triggers with except", :db do
+  include_context "cleanup migrations"
+
+  before(:all) do
+    @old_post = Post.create!(title: 'First', rating: 100, active: true)
+    Dir.chdir("#{File.dirname(__FILE__)}/../dummy") do
+      successfully "rails generate logidze:install"
+      successfully "rails generate logidze:model post --limit=4 --except='created_at,rating' --backfill"
+      successfully "rake db:migrate"
+
+      # Close active connections to handle db variables
+      ActiveRecord::Base.connection_pool.disconnect!
+    end
+
+    Post.reset_column_information
+    # For Rails 4
+    Post.instance_variable_set(:@attribute_names, nil)
+  end
+
+  after(:all) { @old_post.destroy! }
+
+  let(:params) { { title: 'Triggers', rating: 10, active: false } }
+
+  describe "backfill" do
```

Comment:
Please, do not duplicate existing tests. Only test the feature you're working on.
If you want to check that the new options don't break existing functionality, just add them to the existing migrations.